Qualification/Input/qualification_plan.json

```diff
@@ -23,15 +23,15 @@
     },
     {
       "Id": "Erythromycin",
-      "Path": "https://raw.githubusercontent.com/Open-Systems-Pharmacology/Erythromycin-Model/v1.2/Erythromycin-Model.json"
+      "Path": "https://raw.githubusercontent.com/Open-Systems-Pharmacology/Erythromycin-Model/v1.3/Erythromycin-Model.json"
```

Comment:
no change in the model snapshot - just wanted to refer the latest released version